### DISCUSSION TITLE
Cannot start a fresh app

### DISCUSSION THREAD

**Discussion by pastinepolenta** [react-static/react-static#1565] [Q&A] [answered]
Getting a packages related error from a fresh install after running `yarn start` or `react-static start`

```
Running plugins...
[❌] Application bundling failed
multi react-hot-loader/patch ../lib/bootstrapPlugins.js ../lib/bootstrapTemplates.js index.js
Module not found: Error: Can't resolve 'react-hot-loader/patch' in '/home/mirko/dev/sitewego/gino/node_modules/react-static/node_modules'
 @ multi react-hot-loader/patch ../lib/bootstrapPlugins.js ../lib/bootstrapTemplates.js index.js main[0]
multi (webpack)-dev-server/client?http://localhost:3000 (webpack)/hot/only-dev-server.js react-hot-loader/patch ../lib/bootstrapPlugins.js ../lib/bootstrapTemplates.js index.js
... <other errors like that one>....
```

node 12
yarn 1.22.10
Ubuntu 20.04

NOTE: same error with yarn 2

**Comment by SleeplessByte** [comment, marked as answer]
Same as #1567 . Let's move any solutions there :)

**Reply by SleeplessByte** [reply]
A fix was released to make the build run again. `react-hot-loader` is a peerDependency which needs to be installed separately, but you can also try `npx react-static create` (REMOVE your globally installed copy of `react-static` first) and try a new template.
